README.rst

```diff
@@ -385,6 +385,32 @@ The cached values are updated automatically to reflect actual data from database
 the duration of caching these objects using
 `OPENWISP_NOTIFICATIONS_CACHE_TIMEOUT setting <#OPENWISP_NOTIFICATIONS_CACHE_TIMEOUT>`_.
 
+**register_notification_cache_update**
+This function is used to register a signal of a model which is being used as an ``actor``, ``action_object``
+and ``target`` objects. As these values are cached for the optimization purpose so their cached values are
+need to be changed when they are changed. You can register any signal you want which will delete the cached value.
+To register a signal you need to include following code in your ``apps.py``.
+
+.. code-block:: python
+
+    from django.db.models.signals import post_save
+    from swapper import load_model
+
+    def ready(self):
+        super().ready()
+
+        # Include lines after this inside
+        # ready function of you app config class
+        from openwisp_notifications.handlers import register_notification_cache_update
+
+        model = load_model('app_name', 'model_name')
+        register_notification_cache_update(model, post_save, signal_uid="post_save")
```

Comment:
ensure quoting style is consistent (single quotes instead of double quotes, or the opposite, look at the rest of the examples in the docs to decide).

Please also change `signal_uid="post_save"` to have a more unique name, otherwise users may be copy/paste this code which is potentially problematic since post_save is too generic to be a uid.

Eg: 

```python
register_notification_cache_update(
    model,
    post_save,
    signal_uid='myapp_mymodel_notification_cache_invalidation'
)
```

Reply:
Okay I will change them. Thanks